js/tests/unit/collapse.js

```diff
@@ -37,7 +37,7 @@ $(function () {
   })
 
   test('should hide a collapsed element', function () {
-    var $el = $('<div class="collapse"/>').bootstrapCollapse('hide')
+    var $el = $('<div class="collapse in"/>').bootstrapCollapse('hide')
```

Comment:
A Collapsible with `.collapse.in` is in the shown/expanded (i.e. non-collapsed) state, so this change is invalid per the name of the test, AFAICT.